lib/update.py

```diff
@@ -115,6 +116,12 @@ def _update_to_one(element):  # noqa: C901
                 element.get_param('satin_column', False) is False and
                 not element.node.style('stroke-dasharray')):
             element.set_param('stroke_method', 'zigzag_stitch')
+        # grid_size was supposed to be mm, but it was in pixels
+        grid_size = element.get_float_param('grid_size', None)
+        if grid_size:
+            size = grid_size / PIXELS_PER_MM
```

Comment:
Hmmm, I got a grid size of `1.3229166666666665` on my test SVG which is, well, _very_ precise.  Let's limit it to 2 decimal places, which is way more precise than any embroidery machine, by adding this line:

```
grid_size = "%.2f" % grid_size
```